packages/auth/src/graphql/schema.graphql

```diff
@@ -46,6 +46,7 @@ type GrantEdge {
 input GrantFilter {
   identifier: FilterString
   state: FilterGrantState
+  finalizationReason: FilterFinalizationReason
```

Comment:
I think we should also add this to the postman example to show that its there

Reply:
I considered that but figured the default get request should be pretty inclusive and not only return some narrow subset of grants. IE, if me make a few grants and run Get Grants it should have results. 

I guess in that spirit I could do something like finalizationReason: notIn: [REVOKED]. A little non-sensical because state isnt filtered to FINALIZED but demonstrates the existence of the filter.

Reply:
I added added a finalizationReason filter so it's now:

```ts

    "filter": {
        "state": {
            "in": ["PROCESSING", "PENDING", "APPROVED", "FINALIZED"]
        },
        "finalizationReason": {
            "notIn": ["REVOKED"]
        }
    }
```

I think postman should document a realistic usage since this is an actual request. We do have docs which outlines all the filter options. A realistic request wouldn't Include all filter options because some conflict or are dependent in some way (`state`, `finalizationReason`). Exhausting all filter options would look something like this and seems more misleading than helpful. It also requires more work to undo the conflicting bits to get a working query when you want to use it


```ts
"filter": {
        "identifier": {
            "in": ["50c0b796-59fc-4878-a3d9-de6cf5485195"] 
        },
        "state": {
            "in": ["PROCESSING", "PENDING", "APPROVED", "FINALIZED"],
            "notIn": ["PROCESSING", "PENDING", "APPROVED", "FINALIZED"]
        },
        "finalizationReason": {
            "in": ["REVOKED", "ISSUED", "REJECTED"],
            "notIn": ["REVOKED", "ISSUED", "REJECTED"]
        }
    }
```

Reply:
I think what you have is reasonable, I agree that listing out everything ends up being a lot